test/comparison/testdata/fbs/property-update/c/ChangeUid.obx.h.expected

```diff
@@ -109,3 +108,71 @@ static void ChangeUid_free(ChangeUid** object) {
     *object = NULL;
 }
 
+/// Insert or update the given object in the database.
+/// @param object (in & out) will be updated with a newly inserted ID if the one specified previously was zero. If an ID 
+/// was already specified (non-zero), it will remain unchanged.
+/// @return object ID from the object param (see object param docs) or a zero on error. If a zero was returned, you can
+/// check obx_last_error_*() to get the error details. In an unlikely event that those functions return no error
+/// code/message, the error occurred in FlatBuffers serialization, e.g. due to memory allocation issues.
+static obx_id ChangeUid_put(OBX_store* store, ChangeUid* object) {
+    flatcc_builder_t builder;
+    flatcc_builder_init(&builder);
+
+    obx_id id = 0;
+    size_t size = 0;
+    void* buffer = NULL;
+    if (!ChangeUid_to_flatbuffer(&builder, object, &buffer, &size)) {
+        // if the FlatBuffers serialization fails, we clear any previous ObjectBox errors, so that when our caller
+        // checks obx_last_error_code() it will be zero in this case.
+        obx_last_error_clear();
+    } else {
+        OBX_box* box = obx_box(store, ChangeUid_ENTITY_ID);
+        if (box != NULL) {
+            id = obx_box_put_object(box, buffer, size, OBXPutMode_PUT);  // 0 on error
```

Comment:
I wondered as well but since this is C I didn't want to add too much unneeded generated code. I can create a ticket and we can add it any time later if/when there's an actual user request for insert/update semantics

Reply:
also, put3 isn't all that nice name since we might also want to let the user provide the flatcc builder...

Reply:
What I was saying was that put3, as in having three arguments, might collide with the function taking flatcc builder as the argument. Let's follow this up in #8